src/main/java/mousio/etcd4j/transport/EtcdNettyClient.java

```diff
@@ -392,7 +413,6 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
   @Override
   public void close() {
     logger.info("Shutting down Etcd4j Netty client");
-    this.timer.stop();
```

Comment:
should't be better to have a "managed" property like for event loop and explicit close the timer if "managed" ?